### ISSUE TITLE
`NODE_ID` is mistaken as a full keyword

### ISSUE THREAD

**Issue by kennytm** [pingcap/tidb#9910]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

Execute the DDL

```sql
CREATE TABLE a (node_id INT);
```

2. What did you expect to see?

The table successfully created.

3. What did you see instead?

Syntax error:

```
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 22 near "node_id int)" 
```

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?

```
Release Version: v3.0.0-beta-242-g0b28f30cf
Git Commit Hash: 0b28f30cfdef7a396c7d5474d5536787e2d00bd0
Git Branch: master
UTC Build Time: 2019-03-19 08:25:08
GoVersion: go version go1.12 darwin/amd64
Race Enabled: false
TiKV Min Version: 2.1.0-alpha.1-ff3dd160846b7d1aed9079c389fc188f7f5ea13e
Check Table Before Drop: false
```

**Comment by kennytm** [comment]
This also affects the release-2.1 branch:

```
Release Version: v2.1.6-17-g41d90ef37-dirty
Git Commit Hash: 41d90ef372cdbd127c17a44f15502b5bb0cfbb79
Git Branch: release-2.1
UTC Build Time: 2019-03-27 08:14:04
GoVersion: go version go1.12.1 darwin/amd64
Race Enabled: false
TiKV Min Version: 2.1.0-alpha.1-ff3dd160846b7d1aed9079c389fc188f7f5ea13e
Check Table Before Drop: false
```

**Comment by zimulala** [comment]
It seems that is caused by  `https://github.com/pingcap/parser/pull/243`.

**Comment by kennytm** [comment]
Fixed by #9917 and #9928.
